Makefile.am

```diff
@@ -387,6 +387,9 @@ run-rpm-tests-only: rpms
 	$(MAKE) TEST_SUITE_LOG=test-suite.log RPM_PATH=$(RPM_BUILD_DIR) \
 			ROOT_ANACONDA_PATH=$(abs_srcdir) TESTS=rpm_tests/rpm_tests.sh check
 
+cockpit-tests: rpms
```

Comment:
`rpms` is a phony target, so this dependency will cause the rpms to be rebuilt  every time you call `make cockpit-tests`. This is rather annoying, but then the whole Makefile has that problem on many rules. So this can be ignored for now, and optimized later.